go/configure/action.yml

```diff
@@ -26,10 +26,17 @@ runs:
     - name: Setting up private modules access
       shell: bash
       run: git config --global url."https://${{ inputs.FLIPPCIRCLECIPULLER_REPO_TOKEN }}:x-oauth-basic@github.com/wishabi".insteadOf "https://github.com/wishabi"
+    - name: Grab buf version
+      if: ${{ hashFiles('.tool-versions') != '' }}
+      run: |
+        BUF_VERSION=$(awk '/^buf[[:space:]]+/ {print $2}' .tool-versions)
+        echo "BUF_VERSION=${BUF_VERSION}" >> $GITHUB_ENV
+      shell: bash
     - name: Setting up buf
       uses: bufbuild/buf-setup-action@v1
       if: "${{ inputs.BUF_BUILD_API_TOKEN != '' }}"
       with:
         github_token: ${{ inputs.FLIPPCIRCLECIPULLER_REPO_TOKEN }}
         buf_user: ${{ inputs.BUF_BUILD_USER }}
         buf_api_token: ${{ inputs.BUF_BUILD_API_TOKEN }}
+        buf_version: ${{ env.BUF_VERSION }}
```

Comment:
If there's nothing specified in the tool-versions file. It'll be an empty string here and fall back.